src/rust/inetstack/protocols/layer4/mod.rs

```diff
@@ -185,8 +186,8 @@ impl Peer {
             Socket::Udp(socket) => self.udp.bind(socket, socket_addr_v4),
         }?;
 
-        if EphemeralPorts::is_private(socket_addr_v4.port()) {
-            self.ephemeral_ports.reserve(socket_addr_v4.port())?;
+        if self.ephemeral_ports.is_private(socket_addr_v4.port()) {
+            let _ = self.ephemeral_ports.reserve(socket_addr_v4.port());
```

Comment:
If `reserve` returns `None`, will this silently allow it? Is this desirable?

Reply:
I assume that this is correct.